### PR TITLE
Add ID annotator

### PR DESCRIPTION
**Pull request by yy6linda**

### CODE REVIEW

**Comment by yy6linda** [comment]
@tschaffter This is ready for review.

**Review by tschaffter — CHANGES_REQUESTED** [review]
- Remove the folder server/openapi_server/test/lib
- Update the content of docker-compose.yml

File: server/openapi_server/__main__.py [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
 
 def main():
-    app.run(port=8080)
+    app = connexion.App(__name__, specification_dir='./openapi/')
+    app.app.json_encoder = encoder.JSONEncoder
+    app.add_api('openapi.yaml',
+                arguments={'title': 'NLP Sandbox ID Annotator API'},
+                pythonic_params=True)
+
+    app.run(port=8080, debug=True)
```
Comment (tschaffter) [inline server/openapi_server/__main__.py]:
```suggestion
    app.run(port=8080, debug=False)
```

**Review by tschaffter — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -179,16 +179,16 @@ more accurate predictions. When this happens, the version of the tool
...
 
-Here is the protocol that we apply to update this example Date Annotator when a
-new release of the Date Annotator specification is available:
+Here is the protocol that we apply to update this example ID Annotator when a
+new release of the ID Annotator specification is available:
 
 1. Create a new branch
 
-       git checkout -b update-to-specification-x.y.z
+       git checkout -b upid-to-specification-x.y.z
```
Comment (tschaffter) [inline README.md]:
```suggestion
       git checkout -b update-to-specification-x.y.z
```

**Review by tschaffter — APPROVED** [review]
This PR looks great! There just one typo that I will fix now before merging. Thanks! 🚀

**Pull request merged**